### PR TITLE
Shell script for creating docs and running tests

### PR DESCRIPTION
**Pull request by joewalker**

### CODE REVIEW

File: mentat.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
```
Comment (ncalexan) [inline mentat.sh]: nit: license header?

File: mentat.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+doc() {
```
Comment (ncalexan) [inline mentat.sh]: Could you extract a list of packages and interpolate it into the two commands?

**Comment by victorporof** [comment]
It would seem that we _do_ have support for `cargo test --all`, so is this PR really necessary now?

**Comment by joewalker** [comment]
I'll probably close this. Travis is failing with:

```
error: no method named `write_pretty` found for type `types::ValueAndSpan` in the current scope
  --> edn/src/pretty_print.rs:84:25
   |
84 |         assert_eq!(data.write_pretty(40, &mut Vec::new()).is_ok(), true);
   |                         ^^^^^^^^^^^^
```

Although it's passing for me.

**Comment by joewalker** [comment]
Worth noting: we don't have the doc build support that this script provided, but that was kind of a nice to have anyway.

**Pull request closed without merging**